packages/components-vue/projects/vue-wrapper/tests/unit/specs/__snapshots__/provider.spec.ts.snap

```diff
@@ -0,0 +1,27 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`PorscheDesignSystemProvider should render components wrapped with nested provider and prefix 1`] = `
```

Comment:
That snapshot does not look nested at all. Perhaps adding divs might help.

Reply:
Does it tho? The purpose is to test if nested provider allow nested prefixing. The Snapshot looks the same in react (only without those spaces).